setup.py

```diff
@@ -70,6 +71,8 @@
 install_requires = [
     'Flask-BabelEx>=0.9.3',
     'Flask>=0.11.1',
+    'six>=1.12.0',
+    'base32-lib>=1.0.0a1'
```

Comment:
We need a full v1 release of base32-lib because pidstore is already released in final version - should be ok to release base32-lib in final as well.

Reply:
Right, it should not depend on an alpha module.